### ISSUE TITLE
Remove duplicate framework linker for UserNotifications in Swift.package file

### ISSUE THREAD

**Issue by SwiftNativeDeveloper** [urbanairship/ios-library#310]
# Preliminary Info

### What Airship dependencies are you using?
N/A

### What are the versions of any relevant development tools you are using?

The linkerSettings is specifying UserNotifications twice for the AirshipBasement target. While likely no issue, this is unnecessary.

```
                linkerSettings: [
                    //Frameworks
                    .linkedFramework("Network"),
                    .linkedFramework("UserNotifications"),
                    .linkedFramework("UserNotifications"),
```

Main branch, 16.3.0 and earlier
https://github.com/urbanairship/ios-library/blob/dff9148a5b4edf66af0c4a6b1d689914203b3943/Package.swift#L56

# Report

### What unexpected behavior are you seeing?
Swift.package file should be clean

### What is the expected behavior?
Swift.package file has duplicate lines

### What are the steps to reproduce the unexpected behavior?
Code inspection

### Do you have logging for the issue?
N/A

**Comment by rlepinski** [comment]
Thanks! We will take care of that in the next release.

**Comment by rlepinski** [comment]
Fixed in 16.4.0
